# An object for the indicated action
# eg: deviceId, loadId, external triggers

# The key of the parameter
# 
# Field Type: Required
string key

# The value of the parameter that belongs to the key
#
# Field Type: Required
string value

